.circleci/config.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       - run:
           name: Deploy to dev
           command: |
-              ./blue-green-deploy.sh $USERNAME_DEV $CF_PASSWORD_DEV dev
+              ./blue-green-deploy.sh $CRT_USERNAME_DEV $CRT_PASSWORD_DEV dev
```

Comment:
Gotcha! So `$CRT_USERNAME_DEV` etc. all need to be set correctly in Circle for the deploy scripts to work properly? I think that would be helpful to note in a code comment.

Reply:
Ok I added a comment to the top of the file and the instructions for adding the vars are already in the readme https://github.com/usdoj-crt/crt-portal#initial-cloudgov-set-up